### DISCUSSION TITLE
Per chip styling in multiselect

### DISCUSSION THREAD

**Discussion by Real-Gecko** [primefaces/community#1148] [PrimeReact] [answered]
Is there a way to add styling to each chip in multiselect depending on option?
![image](https://github.com/primefaces/community/assets/2231969/0c3b8eec-9205-4c5f-83ea-bdb9775a7a26)
To make something like this possible
![image](https://github.com/primefaces/community/assets/2231969/398347c9-48b2-49a3-916d-0eb5713c9fee)
Or like this
![image](https://github.com/primefaces/community/assets/2231969/370aa3a9-5160-4979-bac3-ef874a360c50)

**Comment by melloware** [comment]
Duplicate question: https://github.com/orgs/primefaces/discussions/803

**Reply by melloware** [reply, marked as answer]
Fixed in 10.5.1+ https://github.com/primefaces/primereact/issues/5935
